packages/block-library/src/navigation/index.php

```diff
@@ -524,18 +518,28 @@ private static function get_responsive_container_markup( $attributes, $inner_blo
 
 		$overlay_inline_styles = esc_attr( safecss_filter_attr( $colors['overlay_inline_styles'] ) );
 
+		// Set the autofocus attribute on the first link.
+		$tag_processor = new WP_HTML_Tag_Processor( $inner_blocks_html );
+		while ( $tag_processor->next_tag( array( 'tag_name' => 'A' ) ) ) {
```

Comment:
This should also look for other focusable elements to correspond with:

https://github.com/WordPress/gutenberg/blob/42d6402af3a85b356390a453689b0adc9d45caf4/packages/block-library/src/navigation/view.js#L158-L165

Only the first of these `focusableSelectors` are currently implemented:

https://github.com/WordPress/gutenberg/blob/42d6402af3a85b356390a453689b0adc9d45caf4/packages/block-library/src/navigation/view.js#L6-L14